src/plugins/data_views/server/rest_api_routes/util/schemas.ts

```diff
@@ -72,5 +72,27 @@ export const runtimeFieldSpec = {
       source: schema.string(),
     })
   ),
+  fields: schema.maybe(
+    schema.recordOf(
+      schema.string(),
+      schema.object({
+        type: runtimeFieldSpecTypeSchema,
+        format: schema.maybe(serializedFieldFormatSchema),
+        customLabel: schema.maybe(schema.string()),
+        popularity: schema.maybe(
+          schema.number({
+            min: 0,
+          })
+        ),
+      })
+    )
+  ),
+  format: schema.maybe(serializedFieldFormatSchema),
+  customLabel: schema.maybe(schema.string()),
+  popularity: schema.maybe(
```

Comment:
I noticed some desync issues with these fields. 

For example:

1. POST runtime_field with `popularity`
2. See `count` is  correct in `fields`, but no `fieldAttrs` info in `data_view` object 


And also: 

1. POST runtime_field with `popularity` 
2. GET data_view - see it in `fieldAttrs` 

```
"fieldAttrs": {
            "composite_runtime.b": {
                "count": 100
            },
            "composite_runtime.a": {}
        },
```

3. DELETE runtime_field
4. GET data_view
5. see `fieldAttrs`
```
"fieldAttrs": {
            "composite_runtime.b": {
                "count": 100
            }
        },
```


I assume we want fieldAttrs cleaned up and not even get desynced?

Reply:
Yes, this needs fixing. Need to add some tests around this.